Source/JavaScriptCore/runtime/StringSplitCache.h

```diff
@@ -30,13 +30,9 @@
 
 namespace JSC {
 
-DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(StringSplitCache);
-
 class JSImmutableButterfly;
 
 class StringSplitCache {
-    WTF_MAKE_FAST_ALLOCATED;
```

Comment:
Ditto StringReplaceCache. Why are you removing the noncopyable/fast malloc macros?

Reply:
Worth noting: this will almost certainly have perf implications

Reply:
Added back.